cli/cmd/plugin/ui/web/tanzu-ui/src/components/LandingPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import styled from 'styled-components';
+ 
+import { CdsButton } from "@cds/react/button";
+import { CdsCard } from "@cds/react/card";
+
+const Section = styled.section`
+    padding: 20px;
+`;
+
+const Header = styled.div`
+    margin-top: 25px;
+    display: flex;
+    align-items: center;
+`;
+
+const Title = styled.span`
+    padding-left: 20px;
+    font-size: 28px;
+`;
+
+const Description = styled.p`
+    padding: 20px;
+    line-height: 24px;
+`;
+
+const SubTitle = styled.h3`
+    padding-left: 20px;
+`
+const ButtonContainer = styled.div`
+    text-align: center;
+    padding-top: 50px;
+`
+const LandingPage: React.FC = () => {
+    const navigate = useNavigate();
+    const cards = ['Docker', 'VMware vsphere', 'Microsoft Azure', 'Amazon EC2'];
+    return (
+        <Section>
+            <Header> 
+                <img src={"tce-logo.png"} alt="Logo"/>
+                <Title>Welcome to the Tanzu Community Edition Launcher</Title>
```

Comment:
I'm wondering if we want to move these TCE-branding strings to another location, but perhaps that's a separate issue?

Reply:
yes. we can do it later when we create some components based on the final version of mockups. In this pr, it mainly shows how we can use state management in react.

Reply:
Raises a good question and I'm thinking that we should address it in a separate issue. I get the sense that we will not have to handle multiple 'brands' for this UI. Will confirm long-term branding roadmap with Josh.